webapp/src/routes/InfoPage/InfoPageDesktop.js

```diff
@@ -335,6 +288,53 @@ const InfoPage = () => {
     }
   }
 
+  useEffect(() => {
+    if (!loadingDataOffer) {
+      const dataOffers = allOffers.offer
+      setOffers(dataOffers)
+      setLoadingOffers(false)
+    }
+  }, [loadingDataOffer, allOffers])
+
+  useEffect(() => {
+
+    if (sponsor_id) {
+      const sponsor = sponsor_id.user[0]
+      setSponsorID(sponsor.id)
+    }
+  }, [sponsor_id])
+
+  useEffect(() => {
+    getInfo()
+    if(profile){
+      if(profile.role === 'sponsor'){
+        getOffers()
+      }
+    }
+  }, [getInfo, getOffers, profile, location])
+
+  useEffect(() => {
+    if (errorUsername && errorInfoProfile) {
+      if (errorUsername.message === 'GraphQL error: Could not verify JWT: JWTExpired') {
+        logout()
+        history.push(`/info/${location.state.profile.account}`)
+      } else history.push('/internal-error')
```

Comment:
Sometimes you use no curly braces when there is just one single statement in the body of a comparison, sometimes not. Maybe we can be consistent and use either one or the other

Reply:
@JustinCast I didn't make that change, I don't really know why is marked as I edited that but It wasn't